### PR TITLE
170111 social search

### PR DESCRIPTION
**Pull request by ewansheldon**
ISSUES-17

### CODE REVIEW

**Comment by simonmorley** [comment]
@ZakMooney Can you merge this please.

File: client/components/audit/social/_index.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
       <br/>
     </form>
     <div flex></div>
+    <range-filter search="test()" ing-if="query.filter || query.start" ></range-filter>
+    <audit-downloads ng-if="query.start && location.id && location_name && guests.length" lid="{{location.id}}" type="guest"></audit-downloads>
```
Comment (simonmorley) [inline client/components/audit/social/_index.html]: Is this right? audit-downloads?

**Comment by ewansheldon** [comment]
sorry - was playing around with that and forgot to remove. should be all good now

**Comment by simonmorley** [comment]
Looks good. Will merge.



On 13 Jan 2017, at 09:32, Ewan Sheldon <notifications@github.com> wrote:

sorry - was playing around with that and forgot to remove. should be all
good now

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/cucumber-tony/cucumber-frontend/pull/474#issuecomment-272399447>,
or mute the thread

**Pull request merged**